### DISCUSSION TITLE
Is it possible to get the value of crosshair horizontal line

### DISCUSSION THREAD

**Discussion by chung362** [tradingview/lightweight-charts#868] [Q&A]
Is it possible to get the value of crosshair horizontal line when using chart.subscribeClick. Or Is it possible to use the ${param.point.y} to calculate the corresponding price?

I want let user adding an alert horizontal line by clicking on the chart.

Thank you~

**Comment by timocov** [comment]
> Is it possible to get the value of crosshair horizontal line when using chart.subscribeClick. Or Is it possible to use the ${param.point.y} to calculate the corresponding price?

No, I don't think so. See #621
